tests/test_recv.py

```
@@ -0,0 +1,172 @@
+		with mock.patch.object(
+			multicast.recv.McastRECV, '_hearstep', return_value=''
+		):
+			self.recv_tool.doStep(is_std=False)
+			# Verify initial debug log and nothing received log
+			mock_logger.debug.assert_any_call("RECV")
+			mock_logger.debug.assert_any_call("Nothing Received.")
+			# Verify that no success log was called
+			mock_logger.info.assert_not_called()
+
+	@mock.patch('multicast.recv.module_logger')
+	def test_doStep_console_output(self, mock_logger):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_doStep_console_output`

Add return type annotation: `None`

(ANN201)